src/modules/plymouthcfg/plymouthcfg.conf

```diff
@@ -1,4 +1,38 @@
 ---
-# The plymouth theme to be set if plymouth binary is present
-# leave commented if packaged default theme should be used
+# Plymouth Configuration Module
+
+# This module can be used to setup the default plymouth theme to 
+# be used with your distribution
+
+# Only set your theme with this module if you are not doing it 
+# already in your operating systems build configurations. 
+
+# You should only use this module if the plymouth package is installed 
+# on the build configurations of your distribution & the plymouth 
+# theme you want to configure is installed as well. 
+
+
+# Leave this commented if you want to use the default theme 
+# shipped with your distribution configurations. 
+
 # plymouth_theme: spinfinity
+
+
+# Change the theme name to the theme you want to use. 
+# Make sure that the theme exists in the themes 
+# directory of plymouth path. 
+
+# If yours is a debian/ubuntu derived distribution, 
+# following are some example plymouth_theme values that might work
+# plymouth_theme: joy 
+# plymouth_theme: script  
+# plymouth_theme: softwaves
+
+# For other distributions you can look under the themes folder for plymouth, 
+# which is usually /usr/share/plymouth/themes and supply any of theme names 
+# from there to plymouth_theme 
+
+# If the plymouth_theme value is left empty or with a theme name which does not exists. 
+# The module would still look for the theme name for availability in plymouth themes folder. 
+# and as theme will not be found, it will leave the plymouth settings unchanged.  
+
```

Comment:
This last paragraph is incorrect: Calamares doesn't check anything: it just edits `plymouthd.conf` and substitutes in the configured theme name. The way to avoid configuring a specific theme (keeping the default) is to leave the setting uncommented. If the theme configured here doesn't exist .. then it's up to plymouth to deal with that.